tests/frontend/tests/test_packages.py

```diff
@@ -0,0 +1,105 @@
+        """
+
+        self.info("Add a package using GitURL")
+        git_url = "https://github.com/threefoldtech/js-sdk/tree/development/jumpscale/packages/notebooks"
+        self.packages.add_package(git_url=git_url)
+
+        self.info("Check that the package has been Added correctly")
+        installed_packages, available_packages = self.packages.check_packages_type()
+        self.assertNotIn("notebooks", available_packages.keys())
+
+        self.info("Add a package using path")
+        path = j.sals.fs.dirname(polls.__file__)
+        self.packages.add_package(path=path)
+
+        self.info("Check that the package has been Added correctly")
+        installed_packages, available_packages = self.packages.check_packages_type()
+        self.assertNotIn("polls", available_packages.keys())
+
+        self.info("Install another package")
+        installed_package = self.packages.install_package()
+
+        self.info("Check that the package has been installed correctly")
+        installed_packages, available_packages = self.packages.check_packages_type()
+        self.assertNotIn(installed_package, available_packages.keys())
+
+        self.info("Delete the three packages")
+        self.packages.delete_package("notebooks")
+        self.packages.delete_package("polls")
+        self.packages.delete_package(installed_package)
+
+        self.info("Check that the packages have been deleted successfully")
+        installed_packages, available_packages = self.packages.check_packages_type()
+        self.assertNotIn("notebooks", installed_packages.keys())
+        self.assertNotIn("polls", installed_packages.keys())
+        self.assertNotIn(installed_package, installed_packages.keys())
+
+    def test03_open_in_browser(self):
+        """
+        Test case for testing open in browser button.
+        **Test Scenario**
+        - Check if threebot deployer package is installed ot not, If not install it.
+        - Press open in browser button.
+        - Check the current URL.
+        """
+        self.info("Press open in browser button")
+        current_url, threebot_installed = self.packages.open_in_browser()
+        if threebot_installed == 1:
+            self.packages.delete_package("threebot_deployer")
+        self.assertEqual(current_url, "https://localhost/threebot_deployer/#/")
+
+    def test04_chatflows(self):
+        """
+        Test case for testing chatflow window.
+        **Test Scenario**
+        - Check if threebot deployer package is installed ot not, If not install it.
+        - Press chatflows button.
+        - Check that the chatflow pop-up window appears.
+        """
+
+        cards_name, threebot_installed = self.packages.chatflows()
+        if threebot_installed == 1:
+            self.packages.delete_package("threebot_deployer")
+        self.info("Press chatflows button")
+        self.info("Check that the chatflow pop-up window appears")
```

Comment:
where is the code that pressing the chatflows button and checking that the pop-up window?